public/i18n/en/translation.json

```diff
@@ -1019,6 +1035,58 @@
       "filter_change_suggestion": "Try changing filter value",
       "copy": "Copy"
     },
+    "firewall_rules": {
+      "title": "Firewall rules",
+      "short_title": "Rules",
+      "forward_rules": "Forward rules",
+      "input_rules": "Input rules",
+      "output_rules": "Output rules",
+      "forward_rules_description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus in lacus nisi. Proin et elit quis nisl interdum iacu",
```

Comment:
Establish rules for data packets moving between zones. Priority is based on rule order, with the first match taking precedence.